docs/UserGuide.md

```diff
@@ -107,8 +111,26 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
     specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
+* `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+
+### Adding favourites: `favourite`
+
+Adds the client to your list of favourite clients and view them.
```

Comment:
Something I encountered in transferring user stories into the DG is that we might be using the terms "person", "contact" and "client" too interchangeably, which might be confusing. I suggest standardising the term for an entry in the app to be "contact", would that be alright?

Reply:
Ok!

Reply:
PS, didn't see the Draft initially.

Reply:
Going to leave this for now in this PR, will change it all in one shot in another PR